app/assets/stylesheets/templates/_home.scss

```diff
@@ -216,45 +216,110 @@
     }
   }
 
-  .slab--testimonials {
-    background-color: white;
-
+  .slab--rating {
+    padding-bottom: 3.5rem;
+    .title {
+      font-size: 2.7rem;
+    }
+    .text--small {
+      font-size: 1.4rem;
```

Comment:
I pulled down the code and this subtitle text looks a little "tight" to the title above it. The line height in figma is set to 20px ... maybe we can try that or add a bit of top spacing?